### ISSUE TITLE
Error occurs when the "arguments" keyword is used inside a curly bracket object

### ISSUE THREAD

**Issue by gogo9th** [babel/babel#12321]
I have the following code which runs fine by NodeJS (save in a file called error.js):

```
'use strict';  
var arr = [];
function a (b, c)
{
   arr.push ({ e: "RESERVED",
               arguments,
               c })
}
a("hi1", "hi2")
a("hi3", "hi4")
console.log(arr)

<OUTPUT>
[ { e: 'RESERVED',
    arguments: { '0': 'hi1', '1': 'hi2' },
    c: 'hi2' },
  { e: 'RESERVED',
    arguments: { '0': 'hi3', '1': 'hi4' },
    c: 'hi4' } ]
```

However, if I transform this with babel by using the code, I get an error:

```
const babel = require('babel-core');
const fs = require('fs')
const code = fs.readFileSync('error.js', 'utf8');
babel.transform(code, {retainLines: true, presets: ['es2017', 'es2015']});


SyntaxError: unknown: arguments is a reserved word in strict mode (8:5)
   6 | {
   7 | 	arr.push ({	e: "RESERVED", 
>  8 | 					arguments,
     | 					^
   9 | 					c })
  10 | }
```

I need to get babel to work on this kind of examples by tomorrow... Is there an easy way to make babel work?

**Comment by JLHwung** [comment]
Duplicate of #10411

**Comment by JLHwung** [comment]
Fixed in `@babel/parser` `v7.11.0`. Please respect the issue template and fill in necessary information, e.g. Babel version.
